packages/react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -134,17 +309,17 @@ const ComboBox = React.forwardRef((props, ref) => {
   }
 
   const filterItems = (items, itemToString, inputValue) =>
-    items.filter((item) =>
+    items.filter((item) => shouldFilterItem ?
       shouldFilterItem({
         item,
         itemToString,
         inputValue,
-      })
+      }) : defaultShouldFilterItem()
```

Comment:
Where is `defaultShouldFilterItem` defined?

Reply:
This is defined here https://github.com/carbon-design-system/carbon/blob/e3823aa2e0b14022df8ba86b91edf61ad6d4948e/packages/react/src/components/ComboBox/ComboBox.tsx#L38